### ISSUE TITLE
normalize data based on training set mean/std

### ISSUE THREAD

**Issue by iamtheeel** [iamtheeel/shake#18]
Do not include the validation set in the norm statistics to avoid information leakage
